src/Paket.Core/Dependencies/PackageResolver.fs

```diff
@@ -246,6 +247,16 @@ module Resolution =
         | ResolutionRaw.ConflictRaw conf -> Conflict conf
     let Ok resolution =
         Resolution.ofRaw [] (ResolutionRaw.OkRaw resolution)
+
+    let fixPackageSettings groupSettings (r:Resolution) =
```

Comment:
needs to be removed (left from the first try to implement a general settings fallback)